app/models/batch_context.rb

```diff
@@ -30,6 +30,7 @@ class BatchContext < ApplicationRecord
   validate :verify_staging_location
   validate :verify_staging_location_path
   validate :verify_file_manifest_exists, if: :using_file_manifest
+  validate :verify_output_dir_no_exists, unless: proc { persisted? }
```

Comment:
This is previously verified in a before_create callback. However, in a testing context when this is tripped, there is no indication what the problem is. This adds it as a normal validation as well.